### DISCUSSION TITLE
"X page found without an <html> element", which page?

### DISCUSSION THREAD

**Discussion by fharper** [11ty/eleventy#3561] [Q&A] [answered]
I have this warning when I generate my website `1 page found without an <html> element` but there is no information about which page it is. I checked my templates, and they all look good, but I have hundreds of pages generated from Markdown, and it's a PITA to check them one by one. I feel like this warning should specify which file is generating the issue, but since it's not, any idea how to easily find the culprit?

**Comment by zachleat** [comment]
I don’t believe I’m aware of that style of warning coming from 11ty? Do you have a test case showing it?

**Reply by zachleat** [reply, marked as answer]
I just ran across this in the wild! Might it be from pagefind? https://pagefind.app

**Reply by fharper** [reply]
You are right. I deactivate the plugin, and didn't get the warning.

So sorry, I usually think about removing plugins before anything... 🤦‍♂️

**Reply by zachleat** [reply]
No worries!

**Comment by fharper** [comment]
Oh, it may be a plugin, but you can check https://github.com/fharper/blog (`yarn` & `yarn dev`). It will display this warning before the `plugin-broken-links` start to share broken links that I need to fix at some point 😅
